fetchai/ledger/contract.py

```diff
@@ -139,7 +133,8 @@ def query(self, api: ContractsApiLike, name: str, **kwargs):
 
         return response['result']
 
-    def action(self, api: ContractsApiLike, name: str, fee: int, *args, signers: Optional[List[Entity]] = None):
+    def action(self, api: ContractsApiLike, name: str, fee: int, signers: List[Entity], *args,
+               from_address: Address = None):
```

Comment:
This is **the most** important change in regards of proper multi-sig support: adding dedicated `from_address` input parameter, which is independent from the `signers`. This enables correct handling of case where deed defines signers which are **different** to address where deed is deployed(the `from_address`).